static/templates/settings/deactivation_stream_modal.hbs

```diff
@@ -4,7 +4,8 @@
         <h3 id="deactivation_stream_modal_label">{{t "Archive stream" }} <span class="stream_name">{{stream_name}}</span></h3>
     </div>
     <div class="modal-body">
-        <p>{{t "Archiving this stream will immediately unsubscribe everyone, and this action cannot be undone." }} <strong>{{t "Are you sure you want to do this?" }}</strong></p>
+        <span>{{t "Archiving stream" }}</span> <strong>{{stream_name}}</strong> <span>{{t "will immediately unsubscribe everyone. This action cannot be undone."}}</span><br>
+        <strong><span>{{t "Are you sure you want to archive this stream?" }}</span></strong>
```

Comment:
![image](https://user-images.githubusercontent.com/59444243/114265179-07243d80-9a0d-11eb-8c54-e111269886a8.png)

Reply:
Here using `<p>` tag breaks the line so I replace it with `<span>` tag.